TriviaNation.Student/Util/RelayCommand.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace TriviaNation.Util
+{
+	public class RelayCommand : ICommand
+	{
+		private readonly Action<object> _Execute;
+
```

Comment:
We should probably move this to shared code, but I can handle that in my branch.